mobilesdk/zcn/transaction_query.go

```diff
@@ -0,0 +1,15 @@
+//go:build mobile
+// +build mobile
+
+package zcn
+
+import (
+	transaction_query_mobile "github.com/0chain/gosdk/zcncore"
```

Comment:
We don't need to alias the package name, it's good to use the default `zcncore`.

Reply:
Okay, I removed alias